### PR TITLE
chore: bump Homebrew formula on new release

### PR DESCRIPTION
**Pull request by vladimyr**
#### Description

Bump Homebrew formula as part of the release workflow.

#### Motivation and Context

Automate all the things 🤖 
Special thanks go to @ATiltedTree who's PR #1360 inspired me to do this.

#### How Has This Been Tested?

- [ ] I have tested using **MacOS**
- [ ] I have tested using **Linux**
- [ ] I have tested using **Windows**

#### Checklist:

- [ ] I have updated the documentation accordingly.
- [ ] I have updated the tests accordingly.

### CODE REVIEW

**Comment by matchai** [comment]
It looks like you might've not noticed this part of our workflow 😅 
https://github.com/starship/starship/blob/master/.github/workflows/workflow.yml#L173-L192

**Comment by vladimyr** [comment]
> It looks like you might've not noticed this part of our workflow 😅

Yeah, it is official, I'm an idiot 🤦 

Let me know what do you think about swapping that snippet with Mislav's action.

File: .github/workflows/workflow.yml [hunk 1]
```diff
@@ -175,21 +175,11 @@ jobs:
     name: Update Brew Formula
     runs-on: macos-latest
```
Comment (matchai) [inline .github/workflows/workflow.yml:176]: It looks like we would no longer need to run this on `macos-latest` with this new action:
```suggestion
    runs-on: ubuntu-latest
```

Comment (vladimyr) [reply .github/workflows/workflow.yml:176]: Indeed 👍

**Pull request merged**